src/cairoUtilFuncGen/inputArgCheck/inputCheck.ts

```diff
@@ -154,7 +152,8 @@ export class InputCheckGen extends StringIndexedFuncGen {
     const funcInfo: GeneratedFunctionInfo = {
       name: funcName,
       code: [
-        `func ${funcName}${IMPLICITS}(arg : ${cairoType.toString()}) -> (){`,
+        `#[implicit(warp_memory)]`,
```

Comment:
There wasn't warp memory in deleted `IMPLICITS`

Reply:
True